### PR TITLE
Added method to set the dimensions of the box

### PR DESCRIPTION
**Pull request by daviesgeek**
I've been using this on a project and this was something I really needed in my project. Not sure if it's super useful here, but I decided to submit a PR anyways.

Usage: `setDimensions({[width: 100], [height: 100]})`

I also added a separate commit in which Sublime trimmed all the trailing whitespace. If that's a problem, I can totally remove it if needed…

### CODE REVIEW

**Comment by heyman** [comment]
Thanks for the PR!

I assume the `[` and `]` in your usage example are typos?

I'd prefer to not merge the whitespace changes, so if you could remove that it would be great! Also, if you want to, you could add a setDimenstions button in the `example/index.html` file, as well as adding a line in the README's example code.

File: src/leaflet-areaselect.js [hunk 1]
```diff
@@ -9,73 +9,84 @@ L.AreaSelect = L.Class.extend({
...
     },
-    
+
+
+    setDimensions: function(dimensions) {
+        if (!dimensions)
+            return
+
+        this._height = dimensions.height || this._height
+        this._width = dimensions.width || this.width
```
Comment (heyman) [inline src/leaflet-areaselect.js]: Hmm, there seems to be a typo here (`this.width` instead of `this._width`)

**Comment by daviesgeek** [comment]
I meant it to be an optional property in the dimensions object. I.e., width and height are optional; one or both can be specified.

I removed the whitespace commit and added an example of `setDimensions()` in the example file

**Comment by heyman** [comment]
Great, thanks!

**Pull request merged**